pyhf/writexml.py

```diff
@@ -1,45 +1,210 @@
+import logging
+
 import os
 import xml.etree.cElementTree as ET
+import numpy as np
+import uproot
+from uproot_methods.classes import TH1
+
+_ROOT_DATA_FILE = {}
```

Comment:
I feel like initializing this as `{}` is misleading since it will become a `TFileRecreate` (which admittedly is dict-like). Since its type will change from dict to TFileRecreate anyways, isn't it better to initialize this as `None`?

Reply:
I can fix this now. I had switched it because I was trying to mock it as a dictionary (since all we'd had ever used it for was dictionary-like access which worked well)